### ISSUE TITLE
Inclusion of steps for Downgrade

### ISSUE THREAD

**Issue by harshild** [motoxplay/stock#16]
This is a frequent scenario where a user is coming from a Custom ROM with Android version 7.X.

We can update the readme for that . If someone has reliable content , i can make changes for the readme

**Comment by brneor** [comment]
The steps to downgrade are usually the same to flash a stock rom, I don't think there's need to create a section specifically for this.

**Comment by harshild** [comment]
Yes this is correct. But when I did the same, for the step to format gpt.bin , remote error was received. If this a general issue so may be we can include a note there .. what are your views on it ?

**Comment by mfonville** [comment]
If anyone has any recommendations / notes ideas, feel free to add them to e.g. the wiki or README.md

**Comment by ccimpoi** [comment]
I tried flashing a 5.x (Germany) over a soft bricked 7.x but it failed with an error that was tied to the device being locked. Same research showed that MotoX does not support downgrade.
I wanted to suggest the README saying that one should not downgrade but the above comment hints that downgrade is actually possible?
